packages/connect-thegraph/src/core/GraphQLWrapper.ts

```diff
@@ -61,7 +34,10 @@ export default class GraphQLWrapper {
     const request = createRequest(query, args)
 
     return pipe(
-      this.#client.executeSubscription(request),
+      this.#client.executeQuery(request, {
+        pollInterval: POLL_INTERVAL,
+        requestPolicy: 'cache-and-network',
```

Comment:
This [request policy](https://formidable.com/open-source/urql/docs/basics/queries/#request-policies) makes urql serve the cache first, and fetch the result again from the network. Network wise it is as efficient as having no cache, but for users, it will make cached queries update instantly, while updating the content right after if the query result has changed.

Reply:
This seems pretty sound; we definitely don't want stale state, but getting instant responses if there's a cache sounds great!

Reply:
Good decisions, being able to update the urlq Client cache on every attempt request would be great for The Graph.